pcsx2/PINE.cpp

```diff
@@ -123,11 +121,13 @@ PINEServer::PINEServer(SysCoreThread* vm, unsigned int slot)
 	// that a "reasonable" value is 5, which is not.
 	listen(m_sock, 4096);
```

Comment:
Lack of error checking here? Although that was a pre-existing issue. But something else listening on the same port should be handled, especially if people run two instances or misconfigure ports.